elements.py

```diff
@@ -1,83 +1,53 @@
 import pygame
 from random import randint
 
-redcircle = pygame.transform.scale(
-    pygame.image.load("assets/redcircle.png"), (40, 40))
-bluecircle = pygame.transform.scale(
-    pygame.image.load("assets/bluecircle.png"), (40, 40))
-redsquare = pygame.transform.scale(
-    pygame.image.load("assets/redsquare.png"), (40, 40))
-bluesquare = pygame.transform.scale(
-    pygame.image.load("assets/bluesquare.png"), (40, 40))
+RED_CIRCLE = pygame.transform.scale(pygame.image.load("assets/redcircle.png"), (40, 40))
+BLUE_CIRCLE = pygame.transform.scale(pygame.image.load("assets/bluecircle.png"), (40, 40))
+RED_SQUARE = pygame.transform.scale(pygame.image.load("assets/redsquare.png"), (40, 40))
+BLUE_SQUARE = pygame.transform.scale(pygame.image.load("assets/bluesquare.png"), (40, 40))
 
-element_list = [redcircle, redsquare, bluecircle, bluesquare]
-x_list = [390, 510, 640, 760]
+ELEMENT_LIST = [RED_CIRCLE, RED_SQUARE, BLUE_CIRCLE, BLUE_SQUARE]
+X_POSITIONS = [390, 510, 640, 760]
 
 
-class Element(object):
+class Element:
 
     def __init__(self):
-        self.x = x_list[randint(0, 3)]
+        self.x = X_POSITIONS[randint(0, 3)]
         self.y = -10
-        self.flag = 0
         self.shape = 0
-        # self.move=1
-        self.blink = 0
-        self.c = 0
-        self.playflag = 0
 
-        if (self.x == 390 or self.x == 510):
-            self.element = element_list[randint(0, 1)]
-            if (self.element == redcircle):
-                self.shape = 1
-            else:
-                self.shape = 2
+        if self.x in (390, 510):
+            self.element = ELEMENT_LIST[randint(0, 1)]
+            self.shape = 1 if self.element == RED_CIRCLE else 2
         else:
-            self.element = element_list[randint(2, 3)]
-            if (self.element == bluecircle):
-                self.shape = 1
-            else:
-                self.shape = 2
+            self.element = ELEMENT_LIST[randint(2, 3)]
+            self.shape = 1 if self.element == BLUE_CIRCLE else 2
 
     def display(self, game):
-        element_rect = self.element.get_rect(center=(
-            self.x + self.element.get_width() / 2,
-            self.y + self.element.get_height() / 2))
-        car1_rect = game.leftcar.get_rect(center=(
-            game.leftcar_x + game.leftcar.get_width() / 2,
-            550 + game.leftcar.get_height() / 2))
-        car2_rect = game.rightcar.get_rect(center=(
-            game.rightcar_x + game.rightcar.get_width() / 2,
-            550 + game.rightcar.get_height() / 2))
+        element_rect = self.element.get_rect(center=(self.x + self.element.get_width() / 2, self.y + self.element.get_height() / 2))
+        car1_rect = game.leftcar.get_rect(center=(game.leftcar_x + game.leftcar.get_width() / 2, 550 + game.leftcar.get_height() / 2))
+        car2_rect = game.rightcar.get_rect(center=(game.rightcar_x + game.rightcar.get_width() / 2, 550 + game.rightcar.get_height() / 2))
```

Comment:
This isn't pep8 compliant so you can leave the old lines and then if you want to change it then do that in another flake8 related PR.